consensus/obcpbft/complainer.go

```diff
@@ -121,12 +121,14 @@ func (c *complainer) SuccessHash(hash string) {
 // The custody queue timeouts are reset.  Restart returns all requests
 // that are maintained in custody.
 func (c *complainer) Restart() map[string]*Request {
-	c.complaints.RemoveAll()
+	complaints := c.complaints.RemoveAll()
 	custody := c.custody.RemoveAll()
 	reqs := make(map[string]*Request)
-	for _, pair := range custody {
-		c.custody.Register(pair.ID, pair.Data)
-		reqs[pair.ID] = pair.Data.(*Request)
+	for _, custodian := range [][]custodian.CustodyPair{complaints, custody} {
```

Comment:
I don't think we should add all our complaints (i.e. received from other replicas) to our custody.